libethereum/Account.cpp

```diff
@@ -81,6 +81,50 @@ PrecompiledContract createPrecompiledContract(js::mObject& _precompiled)
 }
 
 }
+namespace
+{
+	string const c_str_wei = "wei";
```

Comment:
I don't using underscores here other than for `c_` prefix. I'd prefer these to be `c_strWei` or just `c_wei` etc.
Also `c_knownAccountFields `